Sources/SwiftlySearch/SwiftlySearch.swift

```diff
@@ -73,12 +73,14 @@ struct SearchBar<ResultContent: View>: UIViewControllerRepresentable {
     class Coordinator: NSObject, UISearchResultsUpdating, UISearchBarDelegate {
         @Binding
         var text: String
+        var updatedText: String
```

Comment:
This is strictly used inside the Coordinator, so it should be made private (and in keeping with my code style, placed after the public vars).

Reply:
Implemented in c45307b788255c318afc8ddbb486eef03e529863